dev/com.ibm.ws.jaxb.tools.2.3/src/com/ibm/ws/jaxb/tools/SchemaGen.java

```diff
@@ -0,0 +1,22 @@
+/*******************************************************************************
+ * Copyright (c) 2012 IBM Corporation and others.
```

Comment:
I think this should be 2018 - even though it was copied from another project where the date was 2012... same with other files in this project.